lib/rspec/rails/configuration.rb

```diff
@@ -117,22 +118,27 @@ def rendering_views?
         # to the bug fix in rspec/rspec-core#2736, note some of these
         # predicates are a bit nonsensical, but they exist for backwards
         # compatibility, we can tidy these up in `rspec-rails` 5.
+        undef :fixture_path? if respond_to?(:fixture_path?)
```

Comment:
I'm on the fence between this and
```ruby
config.singleton_class.class_eval { attr_accessor :fixture_path }
# ...
def fixture_path?
  !!fixture_path
end
```

Reply:
I'm not, I want to stick with `add_setting` as much as possible, these work arounds will be removed in the next version major version of rspec-rails.